modules/react-native-status/desktop/CMakeLists.txt

```diff
@@ -16,16 +16,30 @@ if (WIN32)
 
   find_package(Go REQUIRED)
 
-  file(STRINGS "../../../STATUS_GO_OWNER" STATUS_GO_OWNER)
   file(STRINGS "../../../STATUS_GO_VERSION" STATUS_GO_VERSION)
-  list(GET STATUS_GO_VERSION -1 STATUS_GO_VERSION)
+  if(STATUS_GO_VERSION MATCHES ";owner=([^;]+).*")
+    set(owner ${CMAKE_MATCH_1})
+  else()
+  set(owner "status-im")
+  message(WARNING "Repository owner name missing from STATUS_GO_VERSION, defaulting to ${owner}")
+  endif()
+  if(STATUS_GO_VERSION MATCHES ";version=([^;]+).*")
+    set(version ${CMAKE_MATCH_1})
+  else()
+    message(FATAL_ERROR "Version name missing from STATUS_GO_VERSION")
+  endif()
+  if(STATUS_GO_VERSION MATCHES ";commit-sha1=([^;]+).*")
+    set(commit ${CMAKE_MATCH_1})
+  else()
+    message(FATAL_ERROR "Commit SHA1 missing from STATUS_GO_VERSION")
+  endif()
```

Comment:
Isn't this overcomplicated? We could turn `STATUS_GO_VERSION` into `status-go-version.json` and just ues https://github.com/sbellus/json-cmake here to load it. Would be cleaner I think.

Reply:
yeah, I guess we could do this with JSON, if bringing an extra dependency to CMake is OK

Reply:
It's just a single file, we can even drop it directly into the repo.